app/services/create_planned_disbursement.rb

```diff
@@ -26,4 +27,10 @@ def call(attributes: {})
   def sanitize_monetary_string(value:)
     Monetize.parse(value)
   end
+
+  def report(activity:)
+    organisation = activity.organisation
+    fund = activity.associated_fund
+    Report.active.find_by(organisation: organisation, fund: fund)
```

Comment:
This is definitely calling out for an association between `Report` and `Fund`, and there are several places where we're associating the two without explicitly calling it such (e.g. the `with_report` trait)

Would lead to being able to do `planned_disbursement.report = activity.report` or even `belongs_to :report, through: :parent_activity` 🥳

Reply:
Yeah we should discuss this with the other devs. Only caveat is that a Fund can (and will) belong to multiple Reports, so we'd need to handle that carefully.

Reply:
Oh, that makes sense. I was confused about associating `Activity` with `Report` because many reports will include entries for the same activity, but a report is definitely scoped to a single fund.

Reply:
I am wary of jumping in here, but yes I agree, in my head fund plays two roles for Report:

1. As a fund is just an activity, it should be associated to the report in which it was created - however as there are only two funds that already exisit, we don't really see this behaviour in production.
2. A report contains only child activities for a given fund i.e. A report is for GCRF activities or Newton activities.

I'll add a card to discuss this next week so we are clear and can write the stories to do the work - I suspect these will be more technical stories (as opposed to user stories) again.